content/general_guidelines/general_guidelines.adoc

```diff
@@ -11,9 +11,26 @@
 == Content
 TBD
 
+=== Documentation
+
+Because you want to keep the image as small as possible, you can avoid installing documentation files along with installing or updating software by specifying the **nodocs** flag. For example:
+
+----
+RUN yum -y update --setopt=tsflags=nodocs
+----
+
 == Updating Base Image Software
-TBD
-// aka yum update in Dockerfiles. Different policy for Fedora and CentOS?
+
+Updating base-image software is a common practice in many Dockerfiles:
+
+----
+RUN yum -y update && yum -y clean all
```

Comment:
Are we really going to have yum update in our Dockerfiles ? I thought we settled that we are going to avoid it.